src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -196,11 +197,13 @@ abstract class SymbolTable extends macros.Universe
   final def pushPhase(ph: Phase): Phase = {
     val current = phase
     phase = ph
-    phStack ::= ph
+    phStack(phStackIndex) = ph
```

Comment:
I was happy to let an `ArrayIndexOutOfBoundException` occur in that case rather than adding explicit code here to handle it.